dispatch/apps/api/views.py

```diff
@@ -403,6 +404,10 @@ def get_queryset(self):
                 Q(description__icontains=q) |
                 Q(host__icontains=q)
             )
+        if pending:
+            queryset = queryset.filter(
+                Q(is_submission__exact=1)
```

Comment:
looks like axel has a test in Event-forms-etc. 
When that's merged I'll pull it into this branch.